apps/academy-wep/components/base/progress-bar.tsx

```
@@ -0,0 +1,98 @@
+            <span>{clampedValue}%</span>
+          </div>
+        )}
+        <div
+          ref={ref}
+          className={cn(progressBarVariants({ size, className }))}
+          role="progressbar"
+          aria-valuenow={clampedValue}
+          aria-valuemin={0}
+          aria-valuemax={100}
+          {...props}
+        >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "progressbar" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "progressbar".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)